central-aggregator-cf-template.yaml

```diff
@@ -384,7 +384,7 @@ Resources:
       Description: Lambda Layer for basic utilities and dependencies used by Support Cases Aggregator related functions.
       ContentUri: layersbuild/
       CompatibleRuntimes:
-        - python3.6
```

Comment:
This layer is still compatible with python3.6 and python3.7 as well as python3.8. Let's have a list of these 3 python types in case future consumers are bound to python3.7 or lower versions and want to reference functions in this layer.